src/controllers/v1/certificate/index.ts

```diff
@@ -302,8 +302,9 @@ export class CertificateController extends Controller {
 
   /**
    * Update a certificate on-chain to include
-   * @summary Create a new demandA on-chain
-   * @param demandAId The demandA's identifier
+   * @summary updates initiated certitificate status to issued on chain
```

Comment:
I think @mattdean-digicatapult fixed this in his previous PR. You may want to rebase